pores4thought/io/BUILD

```diff
@@ -0,0 +1,15 @@
+load("@python_deps//:requirements.bzl", "requirement")
+
+py_test(
+    name = "read_image_test",
+    srcs = ["read_image_test.py"],
+    main = "read_image_test.py",
+    deps = [
+        requirement("numpy"),
+        requirement("pillow"),
+    ],
+    data = [
```

Comment:
Test has a data dependency. In this case it's the S3 data, which it will fetch automatically